modules/polygon/src/polygon-utilities.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable max-statements, max-depth, complexity, no-unused-expressions */
+
+import {equals} from '@math.gl/core';
+
+/** @type {typeof import('./polygon-utilities').WINDING_CLOCKWISE} */
+export const WINDING_CLOCKWISE = 1;
+/** @type {typeof import('./polygon-utilities').WINDING_COUNTER_CLOCKWISE} */
+export const WINDING_COUNTER_CLOCKWISE = -1;
+
+/** @type {typeof import('./polygon-utilities').ensurePolygonWindingDirection} */
+export function ensurePolygonWindingDirection(points, direction) {
```

Comment:
Maybe call this `modify` to emphasize that this function (in contrast to the others) mutates its input.